docs/content/manual/v1.6/manual.yml

```
@@ -332,7 +332,7 @@ sections:
       - title: "Optional Object Identifier-Index: `.foo?`"
         body: |
 
-          Just like `.foo`, but does not output an error even when `.`
+          Just like `.foo`, but does not output even an error when `.`
```

I'm a native speaker and this construction is lousy. Just dropping the word would be clearer

```suggestion
          Just like `.foo`, but does not output an error when `.`
```

---

Done 87c657ddc5a1ffaae866493b638f6ca7cc3b1c37